### PR TITLE
Update record revealed styles, fix some buttons

### PR DESCRIPTION
**Pull request by ahosgood**
Ticket URL: https://national-archives.atlassian.net/browse/UN-802

## About these changes

- Update "Why this record matters" section - removed expandable content and fix padding above and below (/explore-the-collection/stories/the-monteagle-letter/)
- Fix "View record details" and "Use this image" buttons at the bottom of "Why this record matters" section
- Fix "Blogs and podcasts" section alignment with grid (/explore-the-collection/stories/letter-by-the-women-workers-at-fords-of-dagenham/)
- Change "Blogs and podcasts" font weight
- Fix "Read more" button in featured article blocks
- Fix "Browse all stories" button in /explore-the-collection/
- Add in analytics tags

### CODE REVIEW

File: templates/includes/record-matters.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {% load records_tags wagtailcore_tags i18n %}
+
 <div class="generic-intro record-matters">
```
Comment (ahosgood) [inline templates/includes/record-matters.html:3]: TODO: Remove this block before merging

**Comment by JohnHeeryTNA** [comment]
**/explore-the-collection/stories/the-monteagle-letter/**
Are these the correct button styles

Screenshot 2023-10-16 at 14 41 58

**explore-the-collection/stories/letter-by-the-women-workers-at-fords-of-dagenham/**
Blue button:

Screenshot 2023-10-16 at 14 43 41

Blogs and Podcasts section
Long titles are going over two lines (Probably a content decision)

Screenshot 2023-10-16 at 14 44 43

**Comment by ahosgood** [comment]
@JohnHeeryTNA - Do you need to rebuild your CSS? My buttons look like intended:

image

**Review by JohnHeeryTNA — APPROVED** [review]
approving changes

**Pull request merged**